docs/src/markdownDocs/react.mdx

```diff
@@ -0,0 +1,125 @@
+
+## Using the Brightlayer UI CLI
+
+The fastest way to start a new Brightlayer UI project is to use the [Brightlayer UI CLI](/development/cli). The CLI will automatically create a starter project for you (using [Create React App](https://github.com/facebookincubator/create-react-app)) with all of the necessary Brightlayer UI dependencies pre-configured. To start a React project:
+
+<CodeBlock sx={{ height: 'auto', fontSize: '14px' }} code={`npx -p @brightlayer-ui/cli blui new react`} />
+
+You will be asked to provide a name for your project and whether you want to use the Brightlayer UI eslint and prettier configuration files.
+
+The Brightlayer UI CLI will automatically include:
+
+-   MUI Component Library and Icons ([@mui/material](https://www.npmjs.com/package/@mui/material) / [@mui/icons-material](https://www.npmjs.com/package/@mui/icons-material))
+-   Brightlayer UI Component Library ([@brightlayer-ui/react-components](https://www.npmjs.com/package/@brightlayer-ui/react-components))
+-   Brightlayer UI Themes ([@brightlayer-ui/react-themes](https://www.npmjs.com/package/@brightlayer-ui/react-themes))
+-   Brightlayer UI Icons ([@brightlayer-ui/icons-mui](https://www.npmjs.com/package/@brightlayer-ui/icons-mui))
+-   Brightlayer UI Colors ([@brightlayer-ui/colors](https://www.npmjs.com/package/@brightlayer-ui/colors))
+-   Open Sans web font ([@fontsource/open-sans](https://www.npmjs.com/package/@fontsource/open-sans))
+-   Brightlayer UI ESLint and Prettier configurations, if selected ([@brightlayer-ui/eslint-config](https://www.npmjs.com/package/@brightlayer-ui/eslint-config) / [@brightlayer-ui/prettier-config](https://www.npmjs.com/package/@brightlayer-ui/prettier-config))
+
+For detailed usage instructions, refer to the [Brightlayer UI CLI documentation](https://www.npmjs.com/package/@brightlayer-ui/cli).
+
+## Using an Existing Project
+
+If you have an existing project and would like to integrate Brightlayer UI, you can follow the manual integration instructions below. If you have not already started your project, we recommend that you scaffold your project with the Brightlayer UI CLI.
+
+<Divider />
+
+# Manual Brightlayer UI Integration
+
+Before taking the following steps, you'll want to make sure that you have an existing application. If you used the Brightlayer UI CLI to start your project, these steps have already been done for you automatically.
+
+## Including the Component Libraries
+
+### MUI
+
+React projects should make use of the [MUI](https://mui.com/) component library, formerly known as Material UI. Our Brightlayer UI themes are designed to work with these components. You'll also want to install the icons.
+
+<CodeBlock
+    sx={{ height: 'auto', fontSize: '14px' }}
+    code={`npm install --save @mui/material @mui/icons-material @emotion/react @emotion/styled
+// or 
+yarn add @mui/material @mui/icons-material @emotion/react @emotion/styled`}
+/>
```

Comment:
Can you split code blocks with the `// or` comment into two separate code blocks? This is for a better copy&paste function in the future.